packages/input-date/package.json

```diff
@@ -33,7 +33,6 @@
     "lion-input-date.js"
   ],
   "dependencies": {
-    "@lion/core": "0.13.6",
```

Comment:
I guess practically speaking this can be removed because you always get `@lion/core` installed if you depend on any `@lion/<package>`, but it would be more technically correct to have it added here because we import directly from `@lion/core`, therefore the best practice is to add it as a real dependency (npm will dedupe)

Reply:
I didn't see the import anywhere, that's why I removed it. It will be loaded by other dependencies, but technically it's not a input-date direct dependency

Reply:
Ah than it's indeed not needed here.